docs/modules/extend/pages/use-cases.adoc

```diff
@@ -57,6 +57,46 @@ The method `stroke_horizontal_rule` draws a horizontal line using the specified
 The method `ink_prose` is provided by Asciidoctor PDF to make writing text to the page easier.
 Finally, the method `convert` will convert and render the Asciidoctor node that is passed to it, in this case a block image.
 
+== Custom document header without a title page
+
+Documents rendered as articles by Asciidoctor PDF don't have a title page by default.
+Instead their document title is shown on the first content page above the actual content.
+
+Other than in articles rendered by Asciidoctor to HTML, and other than when rendered with a title page (e.g. document type book),
+the author, date and revision remark won't be shown by default.
+To get them included anyway, we can hook into the method `ink_general_heading`, which renders the document title.
+As `ink_general_heading` is also responsible for rendering other "headers" (e.g. section titles),
+we let it fall back to the default implementation for all headers that aren't the document title.
+
+.Extended converter with author, date and revision remark below the document title of document types without a title page
+[,ruby]
+----
+include::example$pdf-converter-article-title-with-author-and-date.rb[]
+----
+<.> Fall back to default implementation unless handling the document title.
+<.> Render the document title.
+See the definition of `ink_document_title` for details.
+<.> Render author, date and revision remark, if they're available.
+See the definition of `ink_document_metainfo` for details.
+<.> Render a vertical gap between the title section (incl. author, date and revision remark) and the body of the document.
+See the definition of `ink_titlesection_body_separator` for details.
+<.> The revision remark isn't available as a field of `doc`, so we have to get it from the hash `doc.attributes`.
+We could do that with `doc.attributes['revremark']`, but we use the convenience method `doc.attr` instead.
```

Comment:
The information about using `doc.attributes` is overstated. It's also not the preferred way to access attributes anyway, so no one should be using that way.

Reply:
I've completely removed mention of `doc.attributes` with bd5563feec8497d4cbda878ce5427d86bb1e629e. Does the resulting callout read alright?